packages/browser-extension/source/util/morphFunctions.ts

```diff
@@ -1,17 +1,13 @@
-import { Project } from "ts-morph"
+import { Project, SourceFile, SyntaxKind } from "ts-morph"
 
 export const addRelativeImportWithMorph = (
-  project: Project,
-  code: string,
+  file: SourceFile,
   newPageName: string
-) => {
-  const alteredFile = project.createSourceFile(`App.tsx`, code)
-  alteredFile.addImportDeclaration({
+) =>
+  file.addImportDeclaration({
     defaultImport: newPageName,
     moduleSpecifier: `./pages/${newPageName}`,
```

Comment:
page path can not be card coded - it is not general enough (different projects use different conventions: routes, containers)